utils.py

```diff
@@ -1,6 +1,9 @@
 import cudf
 import cupy as cp
 
+# reproducibility
+cp.random.seed(seed=0)
```

Comment:
Should this line be in `conftest.py` instead?

Reply:
I'd rather not put something like this into a global file modifying all state. The proper solution here (and throughout benchmarks) would be to use [cupy's RandomState](https://docs.cupy.dev/en/stable/reference/generated/cupy.random.RandomState.html) object to provide a local reproducible context where random numbers are generated. If you feel up for making that change, please do.